code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -527,6 +527,7 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT|HIDETAIL
 	flags_size = ONESIZEFITSALL
 	strip_delay = 60
+	breakouttime = 3000

```

Comment:
Just a request, but for those of us who are stupid could you put a comment after here that says how long it is in real time? How many seconds/minutes?